### DISCUSSION TITLE
Slight change in the slide

### DISCUSSION THREAD

**Discussion by AVI18794** [mrdbourke/pytorch-deep-learning#75] [General]
@mrdbourke  While explaining the concepts what is neural network, there is a slight typo in the  slide  in both the tensorflow version and pytorch version. At page 15 in the label where it is supposed to be a label ' Not a disaster' it is mentioned as 'Not a diaster'.
Thanks will be going through  the video and report them if I found anything else in the slides/videos/file.
